### ISSUE TITLE
Hi? Do u know why I can only download about 180w samples? where are others, it's my network problem or other problem?

### ISSUE THREAD

**Issue by yaorong1996** [igorbrigadir/DownloadConceptualCaptions#7]

**Comment by igorbrigadir** [comment]
Is this for Training set or Validation set? How many exactly did you manage to download? It sounds like a network error - maybe you lost connectivity and they all timed out, i don't think that many got deleted already. 

Have a look at the output of https://github.com/igorbrigadir/DownloadConceptualCaptions/blob/master/downloaded_report.ipynb (you'll need to re-run it all using your own `downloaded_training_report.tsv.gz` for example) that report should give you a breakdown of HTTP Status codes that may give you an answer to what went wrong

**Comment by MartinPernus** [comment]
I seem to have a similar problem, I can only download 20 images from the validation set, the majority of other HTTP requests returns code 408 Request Timeout, even with only 1 process. Do you have any idea what the problem could be?

**Comment by igorbrigadir** [comment]
Re-open and let me know if it's still a problem!
